let a=prompt("Enter The Number:- ")
let b=prompt("Enter The Divident Number:- ")
try{
    console.log(a/b)
    dsds
}
catch(error){
    console.log("You have generated some error the error is:- "+error)
}


/*you cannot handle the error like this */
/*try{
    setTimeout(()=>{
        //console.log(a/b)
        dsds
    },5000)
}
catch(error){
    console.log("You have generated some error the error is:- "+error)
}*/



